install-monorepo.sh

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env bash
+
+set -o pipefail
+
+BASE=$(cd $(dirname "${BASH_SOURCE[0]}")/.. && pwd)
+
+# Print help and exit.
+function usage {
+	cat <<-EOH
+		usage: $0 
+
+		Installs all the required tooling for the Jetpack Monorepo. 
+	EOH
+	exit 1
+}
+
+if [[ $1 ]]; then
+	usage
+fi
+
+# Check if we're on a Mac or Linux, bail if we're not.
+OS="$(uname)"
+if [[ "$OS" == "Linux" ]]; then
+	ON_LINUX=1
+	sudo apt install build-essential
+elif [[ "$OS" != "Darwin" ]]; then
+	abort "Installer script is only supported on macOS and Linux."
+fi
+
+# Check of Homebrew and nvm are installed
+echo "Checking if Homebrew is installed..."
+if ! command -v brew &>/dev/null; then
+    echo "Installing Homebrew"
+    /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+	if [[ -n "$ON_LINUX" ]]; then # Add homebrew to PATH
+	# Reset the terminal so it picks up the changes.
+
+		echo 'eval "$("$HOME/.linuxbrew/bin/brew" shellenv)"' >> "$HOME/.profile"
+		eval "$("$HOME/.linuxbrew/bin/brew" shellenv)"
+		PATH="$HOME/.linuxbrew/bin:$PATH"
+			if [[ "$SHELL" == "/bin/zsh" ]]; then
+		echo "Refreshing terminal"
+		source ~./zshrc
+	elif [[ "$SHELL" == "/bin/bash" ]]; then
+		echo "Refreshing terminal"
+		source ~./bashrc
+	fi
+	fi
+else
+	echo "Updating brew"
+    brew update
+	# Brew can be finicky on MacOS
+	if [[ $? -ne 0 ]]; then
+		 echo "Reinstalling Homebrew"
+   	 	/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+	fi
+fi
+
+echo "Checking if NVM is installed..."
+if ! command -v nvm &>/dev/null; then
+    echo "Installing nvm"
+    curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.0/install.sh | bash && export NVM_DIR=$HOME/.nvm && source $NVM_DIR/nvm.sh
```

Comment:
Looks like this will fix nvm erroring out.
```suggestion
	curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.0/install.sh | bash && export NVM_DIR=$HOME/.nvm && source $NVM_DIR/nvm.sh --no-use
```
Apparently by default it tries to do `nvm use` without first doing `nvm install`, and it suppresses the actual output from `nvm use` too.